src/table/cells/handle-cell-focus.js

```diff
@@ -0,0 +1,25 @@
+export const focusCell = (rowElements, nextCellCoord) => {
+  const nextCell = rowElements[nextCellCoord[0]].getElementsByClassName('sn-table-cell')[nextCellCoord[1]];
+  nextCell.focus();
+  nextCell.setAttribute('tabIndex', '0');
+};
+
+const removeTabSequence = (rowElements, nextCellCoord) => {
```

Comment:
This name is a bit misleading. Use `removeTabStop` or `resetTabIndex` or something

Reply:
sure. I will go for resetTabIndex